velox/connectors/hive/TableHandle.h

```diff
@@ -115,6 +129,9 @@ class HiveColumnHandle : public ColumnHandle {
   const TypePtr dataType_;
   const TypePtr hiveType_;
   const std::vector<common::Subfield> requiredSubfields_;
+  // The column parameters are used for metadata like Iceberg date partition
+  // value format.
+  const std::unordered_map<std::string, std::string> columnParameters_;
```

Comment:
Let's just make it typed struct as there is no need for serialization for this parameter.  Something like this:
```c++
struct ColumnParseParameters {
  enum PartitionDateValueFormat {
    kISO8601,
    kDaysSinceEpoch,
  } partitionDateValueFormat;
};
```

Reply:
Thanks @Yuhta . I have made the changes. Also, the draft changes on Prestissimo side are at https://github.com/nmahadevuni/presto/commit/32b22cbea07491f03306029913b4b97769937298. Please review @majetideepak @aditi-pandit